lib/Sema/ImportResolution.cpp

```diff
@@ -574,8 +574,14 @@ ModuleImplicitImportsRequest::evaluate(Evaluator &evaluator,
       !clangImporter->importBridgingHeader(bridgingHeaderPath, module)) {
     auto *headerModule = clangImporter->getImportedHeaderModule();
     assert(headerModule && "Didn't load bridging header?");
-    imports.emplace_back(
+    AttributedImport<ImportedModule> import(
         ImportedModule(headerModule), SourceLoc(), ImportFlags::Exported);
+    if (ctx.ClangImporterOpts.BridgingHeaderIsInternal) {
+      import.accessLevel = AccessLevel::Internal;
+      import.options |= ImportFlags::ImplementationOnly;
```

Comment:
The mix of `@_implementationOnly internal import` isn't exactly an expected configuration at the moment. I'm worried that checks for both could overlap and duplicate the exportability diagnostics. We could use only `internal import` for library-evolution enabled modules if it's an issue.

Do we still see the warnings on implementation-only use from non-library-evolution modules for an implicit import like this?